gridscale/server_concurrency_handler.go

```diff
@@ -74,7 +76,11 @@ func (l *serverStatusList) removeServerSynchronously(ctx context.Context, c *gsc
 			//set the shutdown timeout specifically
 			shutdownCtx, cancel := context.WithTimeout(context.Background(), serverShutdownTimeoutSecs*time.Second)
 			defer cancel()
-			err := c.ShutdownServer(shutdownCtx, id)
+			err := errHandler.SuppressHTTPErrorCodesWithSubErrString(
+				c.ShutdownServer(shutdownCtx, id),
+				serverAlreadyInRequestedPowerStateErrSubStr,
+				http.StatusBadRequest,
```

Comment:
U do not need to pass `http.StatusBadRequest` as an argument. U can use it directly in `SuppressHTTPErrorCodesWithSubErrString`